.github/workflows/release.yml

```
@@ -432,6 +433,7 @@ jobs:
         uses: softprops/action-gh-release@v1
         with:
           name: Android AAB Release - ${{ github.ref_name }}
+          tag_name: ${{ steps.get_version }}
```

actionlint (1.7.7)

object, array, and null values should not be evaluated in template with ${{ }} but evaluating the value of type {conclusion: string; outcome: string; outputs: {ruby-prefix: string}}

(expression)